### PR TITLE
Updates glare to python 3

### PR DESCRIPTION
**Pull request by bp-kelley**
Updates the Glare algorithm to python 3 add slightly better docs.

### CODE REVIEW

File: Contrib/Glare/glare.py [hunk 1]
```diff
@@ -132,7 +146,7 @@ def prune(self, fractionToKeep):
         fractionToKeep percentage"""
         assert 0 < fractionToKeep <= 1.0, "fractionToKeep: %s"%fractionToKeep
 
-        self.sidechains.sort(lambda x,y: -cmp(x.good_count, y.good_count))
+        self.sidechains.sort(key=lambda x: x.good_count, reverse=True)#,y: -cmp(x.good_count, y.good_count))
```
Comment (greglandrum) [inline Contrib/Glare/glare.py:149]: might as well remove the commented out part

File: Contrib/Glare/glare.py [hunk 2]
```diff
@@ -166,13 +180,13 @@ def getSidechainsPerPartition( self, total_num_partitions_per_rgroup ):
...
-        sizes = [ (libIdx, max(rg.count()/total_num_partitions_per_rgroup, 1))
+        sizes = [ (libIdx, max(rg.count()//total_num_partitions_per_rgroup, 1))
                   for libIdx, rg in enumerate(self.rgroups) ]
 
-        # "optimially" apportion the partitions according the
+        # "optimally" apportion the partitions according the
         #  the glare paper see Appendix eq (8) and (9)
         # sort by size
-        sizes.sort(lambda x,y: cmp(x[1], y[1]))
+        sizes.sort(key=lambda sz: sz[1])#cmp(x[1], y[1]))
```
Comment (greglandrum) [inline Contrib/Glare/glare.py:189]: might as well remove the commented out part

**Review by greglandrum — CHANGES_REQUESTED** [review]
just a couple of cleanup suggestions

**Comment by bp-kelley** [comment]
@greglandrum I accidentally deleted the source branch, I'm pushing a new PR with the changes

**Pull request closed without merging**